README.md

```diff
@@ -14,8 +14,8 @@ In order to speed up this port, we are running the first version using the origi
 
 You'll need some basic requirements installed on your machine, probably through your package manager:
 
-- Python 3.8.x
-- Node.JS 7+
+- Python 3.12.x
+- Node.js
```

Comment:
Can we update the `engines` declaration in package.json to

```
"engines": {
  "node": ">=18"
}
```

Reply:
Fixed with https://github.com/DemocracyClub/electionleaflets/pull/255/commits/d57846a4c2f7cac802d1a2d77149b5c67d8d73c9

Reply:
https://github.com/DemocracyClub/electionleaflets/pull/255#discussion_r1775428093